package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dash-bio",
-  "version": "0.1.2rc5",
+  "version": "0.1.2rc6",
```

Comment:
These pesky character version strings make R unhappy, but that's my problem and not yours. Release candidates are a thing, R people!